db/migrate/20230530025604_create_standup_meetings.rb

```diff
@@ -0,0 +1,17 @@
+class CreateStandupMeetings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :standup_meetings do |t|
+      t.references :standup_meeting_group, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
+
+      t.text :yesterday_work_description, null: false
+      t.text :today_work_description, null: false
+      t.text :blockers_description, null: false
```

Comment:
Curious if you had any thoughts about how the user will declare that they're not blocked. I guess the intent here might be to have it work like the DailyBot where you have the user put in "no" or "none" or something?

Reply:
That was my thought initially but now that I am thinking about it again we should just make this nullable.

Reply:
Just updated to be nullable. It would also make querying easier because we just check if blockers is not null. We will need to have a tooltip or something to tell the user to leave blank (or maybe just utilize form to show it better)

Reply:
Yeah that's what I was thinking. A nullable field would be pretty convenient to query against for this purpose, so I definitely agree with the change.